test/sqllogictest/mz-resolve-object-name.slt

```diff
@@ -229,6 +229,12 @@ s86 1397
 s86 1398
 s86 1705
 
+query T
+SELECT DISTINCT mz_internal.mz_global_id_to_name(id) AS name
```

Comment:
They work on two different levels; the `DISTINCT` in the function ensures that the query returned from `mz_objects` returns only one row––without that, it returns the name for each matching `GlobalId`, which (I had forgotten) is not unique. This is invalid for an expression in the scalar position.

The `DISTINCT` here is because otherwise we'd receive the name for each entry in `mz_objects`, even though they're all the same. It looked messy to have all of them.